### PR TITLE
Enable fio action pages work on load from email link

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: client/src/components/FioNameRenew/components/RenewEdgeWallet.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  const renew = async ({ keys, data }: SubmitActionParams) => {
+    const { name } = data;
+    if (isDomain(name)) {
+      return await apis.fio.executeAction(keys, ACTIONS.renewFioDomain, {
+        fioDomain: name,
+        maxFee: fee,
+      });
+    }
+
+    throw new Error("Can't renew FIO Crypto Handle");
```
Comment (trukhilio) [inline client/src/components/FioNameRenew/components/RenewEdgeWallet.tsx:45]: Do we need this phrase because we don't renew crypto handle anymore?

Comment (andreyvEze) [reply client/src/components/FioNameRenew/components/RenewEdgeWallet.tsx:45]: yes

File: server/src/jobs/wallet-data.mjs [hunk 2]
```diff
@@ -178,7 +186,7 @@ const checkBalance = async wallet => {
     if (!new MathOp(publicWalletData.balance).eq(balance)) {
       const roe = await getROE();
       const fioNativeChangeBalance = new MathOp(balance)
-        .div(publicWalletData.balance)
+        .sub(publicWalletData.balance)
```
Comment (trukhilio) [inline server/src/jobs/wallet-data.mjs:189]: handle 0 balance situation

File: server/src/jobs/wallet-data.mjs [hunk 3]
```diff
@@ -37,7 +37,11 @@ const postMessage = message => {
 };
 
 const logFioError = e => {
-  if (e.errorCode !== 404) logger.error(e);
+  try {
```
Comment (trukhilio) [inline server/src/jobs/wallet-data.mjs:40]: remove try catch. Just check `e`

File: client/src/pages/FioRequestDecryptPage/FioRequestDecryptPage.tsx [hunk 4]
```diff
@@ -0,0 +1,292 @@
...
+
+  // todo: check if fio address from request is in wallet addresses
+
+  useEffect(() => {
+    setDecryptData(null);
+  }, []);
+
+  useEffect(() => {
+    if (fioRequestId && fioWalletsData && publicKey && !fioRequest) {
+      const { receivedFioRequests, sentFioRequests, obtData } = fioWalletsData[
```
Comment (trukhilio) [inline client/src/pages/FioRequestDecryptPage/FioRequestDecryptPage.tsx:74]: check if wallet is on another account

File: client/src/pages/FioRequestDecryptPage/FioRequestDecryptPage.tsx [hunk 5]
```diff
@@ -0,0 +1,292 @@
...
+        publicKey
+      ];
+      const receivedRequest = receivedFioRequests.find(
```
Comment (trukhilio) [inline client/src/pages/FioRequestDecryptPage/FioRequestDecryptPage.tsx:77]: Crashes if `null`

**Review by trukhilio — COMMENTED** [review]
Check comments

**Pull request merged**